tests/rest/client/test_relations.py

```diff
@@ -981,6 +981,28 @@ def test_background_update(self) -> None:
             [annotation_event_id_good, thread_event_id],
         )
 
+    def test_bundled_aggregations_with_filter(self):
+        """If unsigned is not a returned field, the bundled aggregations still get added."""
```

Comment:
I would remark in this test description that this is a design choice for ease and not something that the spec requires (indeed, maybe one day it will make sense to save the bytes and leave it off).